content/fr/contribution-guide/index.md

```diff
@@ -1,105 +1,105 @@
 ---
 template: post
-title: Contribution Guide
-description: Any contribution to Nuxt is more than welcome!
+title: Guide de contribution
+description: Toute contribution à Nuxt est plus que bienvenue !
 back: false
 ---
 
-> Any contribution to Nuxt is more than welcome!
+> Toute contribution à Nuxt est plus que bienvenue !
 
-## Reporting Issues
+## Signaler des erreurs
 
-A great way to contribute to the project is to send a detailed report when you encounter an issue: [Bug report](https://github.com/nuxt/nuxt.js/issues/new?assignees=&labels=bug-report&template=bug-report.md&title=)
+Une excellente façon de contribuer au projet est d'envoyer un rapport détaillé lorsque vous rencontrez un problème : [Bug report](https://github.com/nuxt/nuxt.js/issues/new?assignees=&labels=bug-report&template=bug-report.md&title=)
 
-Please make sure to include a reproduction repository or [CodeSandBox](https://template.nuxtjs.org/) so that bugs can be reproduced without great efforts. The better a bug can be reproduced, the faster we can start fixing it!
+Veuillez vous assurer d'inclure un dépôt de reproduction ou [CodeSandBox](https://template.nuxtjs.org/) afin que les bogues puissent être reproduits sans grands efforts. Plus un bug peut être reproduit, plus vite nous pouvons commencer à le corriger !
 
 ## Pull Requests
 
-We'd love to see your pull requests, even if it's just to fix a typo!
+Nous serons ravis de recevoir vos demandes de modification, même s'il s'agit simplement de corriger une coquille !
 
-However, any significant improvement should be associated to an existing [feature request](https://feature.nuxtjs.org/) or [bug report](https://bug.nuxtjs.org/).
+Cependant, toute amélioration significative doit être associée à une demande de [fonctionnalité](https://feature.nuxtjs.org/) ou d'un [bug report](https://bug.nuxtjs.org/).
 
 ### Getting started
```

Comment:
I don't know the best words to translate this but something like "Démarrer"/"Bien démarrer"?